js/MVP/views/moteinnkallelse/Moteinnkallelse.js

```diff
@@ -76,6 +77,11 @@ const Moteinnkallelse = () => {
 
   const { tid, uuid, brevType, document, lestDato } = data[0];
 
+  if (brevType === brevTypes.AVLYST && lestDato === null) {
+    postLestBrev(uuid).then((r) => console.log(`Avlysning les status: ${r}`));
```

Comment:
Foreslår å bruke useMutatebrevLest for å oppdatere, så får vi react query håndtering av server state. Og kanskje fjerne console.log for prod?